Plots/XY/NCL_tm_2.py

```diff
@@ -0,0 +1,113 @@
+"""
+NCL_tm_2.py
+===============
+Tickmark and Axis Manipulation example
+
+This Python script reproduces the NCL plot script found here: https://www.ncl.ucar.edu/Applications/Scripts/tm_2.ncl
+"""
+
+
+###############################################################################
+# Import the necessary python libraries
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.ticker import (MultipleLocator, FormatStrFormatter)
+
+###############################################################################
+# Define a function for creating an array of random values.
+
+
+def random_floats(low, high, size):
+    values = [np.random.uniform(low, high) for _ in range(size)]
+    return values
```

Comment:
This function is unnecessary.  The `np.random.uniform` function actually has the signature `(low, high, size)` and returns a NumPy array with length `size`.

Reply:
Should be fixed in latest push.